session.cpp

```diff
@@ -54,54 +60,155 @@ int session::handle_request(const boost::system::error_code& error,
 
 }
 
+// setup static file serving response
+int session::handle_static(std::string& url, http_response& response) {
+    std::cout << "========= Handling Static =========" << std::endl;
+    
+    std::string abs_path = get_exec_path() + "/public" + get_path_from_url(url);
+    std::cout << "Serving file from: " << abs_path << std::endl;
+    
+    if (!file_exists(abs_path)) {
+      // TODO: 404 Error
+      std::cerr << "Error: " << abs_path << " does not exist" << std::endl;
+      return -1;
+    } 
+
```

Comment:
Do we have a class to send error messages back to the browser (404 error, etc)? I don't know if we need that for now, but it is definitely something we will need for later.